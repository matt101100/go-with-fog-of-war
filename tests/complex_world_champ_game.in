place H8
place I7
place K11
place G7
history
place J7
place H6
who
place J8
place G6
place J6
place J9
place I5
place F6
place I6
place G5
who
place G4
place G8
place G9
place K8
place J5
place J4
place L5
place F4
place E3
place K5
place G3
view
place H4
place F3
place D6
place H5
