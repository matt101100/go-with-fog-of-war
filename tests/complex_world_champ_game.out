H8I7K11G7
B
B
E1,....o#....#.#...............xxxxxxxxxxxxxxxxxxxxx
Black wins!
H8I7K11G7J7H6J8G6J6J9I5F6I6G5G4G8G9K8J5J4L5F4E3K5G3H4F3D6H5
Thank you for playing!
